### DISCUSSION TITLE
Custom Font Issue

### DISCUSSION THREAD

**Discussion by ghost** [community/community#143989] [Mobile]
Eversince I downloaded this GitHub mobile app I've been having problems with the font like it seems the app is adapting to the phone's font and not the default app font. And I am having problems adaptating to it. I really prefer the app Custom Font please help me rectify it. 

P.S I prefer the font of the second screenshot because it's the app's custom font I think.

![Screenshot_20241108-175348_GitHub.jpg](https://github.com/user-attachments/assets/467c2f1f-d108-40e1-9018-09e08d5d97e6)

![Screenshot_20241108-175358_GitHub.jpg](https://github.com/user-attachments/assets/fb5acf9d-a45b-48fc-9916-3ea9016a106c)

**Comment by marlonwq** [comment]
Yes, the app adapts to the phone's default font, I don't know of any configuration option to customize the font in the github mobile app.

The font of the first screenshot is cool, but anyway I don't think it's necessary to change it, because in most cases the github mobile app is only used to authorize login, allowing you to read what is written is not that relevant changing the font Haha
